src/dtgtk/thumbtable.c

```diff
@@ -1129,6 +1129,8 @@ static gboolean _event_enter_notify(GtkWidget *widget,
                                     GdkEventCrossing *event,
                                     gpointer user_data)
 {
+  darktable.view_manager->proxy.activity.last_time = dt_get_wtime();
```

Comment:
Maybe a single assignment in `_event_draw()` in `thumbtable.c`?

Reply:
Good point...

Reply:
done

Reply:
Reverted the single assigment as that interferes with lighttable actions done elsewhere ...